nntrainer/tensor/manager.h

```diff
@@ -349,6 +396,8 @@ class Manager {
    * @param[in] is_weight true if weight, else false meaning its gradient
    */
   AllocFunc getAllocFunc(bool is_weight);
+
+  void deallocateTensorsButWeights();
```

Comment:
Could you check if this function is used / implemented? I might have missed :(

Reply:
removed.